rest_framework/locale/ko_KR/LC_MESSAGES/django.po

```diff
@@ -1,37 +1,43 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
-# 
+#
 # Translators:
-# GarakdongBigBoy <novelview9@gmail.com>, 2017
+# JAEGYUN JUNG <twicegoddessana1229@gmail.com>, 2024
 # Hochul Kwak <rhkrghcjf@gmail.com>, 2018
+# GarakdongBigBoy <novelview9@gmail.com>, 2017
 # Joon Hwan 김준환 <xncbf12@gmail.com>, 2017
 # SUN CHOI <best2378@gmail.com>, 2015
 msgid ""
 msgstr ""
 "Project-Id-Version: Django REST framework\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-10-13 21:45+0200\n"
+"POT-Creation-Date: 2024-10-22 16:13+0900\n"
 "PO-Revision-Date: 2020-10-13 19:45+0000\n"
-"Last-Translator: Xavier Ordoquy <xordoquy@linovia.com>\n"
-"Language-Team: Korean (Korea) (http://www.transifex.com/django-rest-framework-1/django-rest-framework/language/ko_KR/)\n"
```

Comment:
Since we decided to handle translations in Github(#9535), i deleted the transifex line.